web/src/main/java/com/marklogic/hub/web/service/DataHubService.java

```diff
@@ -187,6 +184,7 @@ private void installUserModules(HubConfig hubConfig, boolean forceLoad, DeployUs
         loadUserArtifactsCommand.setHubConfig(hubConfig);
         loadUserArtifactsCommand.setForceLoad(forceLoad);
 
+        // TODO Why load hub artifacts when this method is for loading user modules/artifacts?
```

Comment:
There is an option to clear all dbs in the QS dashboard. If someone clicks that, the default flows and step definitions will be deleted as well. This is meant to reload them